.github/workflows/command-block.yml

```diff
@@ -4,9 +4,16 @@ on:
     paths:
       - 'commands/docs/**'
 
+# Fix Resource not accessible by integration (updatePullRequest)
+permissions:
+  contents: write
```

Comment:
Thanks for digging into why that workflow never commented!

Do we need the `contents` permissions, here? I think those are for modifying the repository itself or comments associated with commits for some reason.

Reply:
I have tried many ways, It works in my repo: https://github.com/hustcer/nushell.github.io/pull/3  however, it does not work here: https://github.com/nushell/nushell.github.io/actions/workflows/command-block.yml
I guess some settings in `Actions permissions` need to be changed and I don't have the permission @fdncred

Reply:
Resolved by #1704 and following